lib/cool.io/dns_resolver.rb

```diff
@@ -41,7 +41,7 @@ class DNSResolver < IOWatcher
     # if it errs due to inability to reach the DNS server [Errno::EHOSTUNREACH], same
     # Query /etc/hosts (or the specified hostfile) for the given host
     def self.hosts(host, hostfile = Resolv::Hosts::DefaultFileName)
-      hosts = {}
+      hosts = {"localhost" => "::1"}
```

Comment:
Use IPv6 is safe on both Linux/Windows?

Reply:
I think so.

* On Linux, empty `/etc/hosts` is a rare case so it will be overridden appropriately the following parts
* On Windows, IPv6 is enabled, and take a priority to IPv4 nowadays

It may be safer to support the case that priority was changed.

Reply:
I've kept  default `hosts` empty and added fallback code instead.
It makes more safer on both Linux/Windows.